src/Sylius/Bundle/CoreBundle/Form/Type/Order/AddToCartType.php

```diff
@@ -12,34 +12,36 @@
 namespace Sylius\Bundle\CoreBundle\Form\Type\Order;
 
 use Sylius\Bundle\OrderBundle\Controller\AddCartItemCommand;
+use Sylius\Bundle\OrderBundle\Controller\AddToCartCommand;
 use Sylius\Bundle\ResourceBundle\Form\Type\AbstractResourceType;
 use Sylius\Component\Core\Model\ProductInterface;
+use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
 /**
  * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
  */
-final class AddToCartType extends AbstractResourceType
+final class AddToCartType extends AbstractType
 {
     /**
      * {@inheritDoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder
-            ->add('cartItem', CartItemType::class, ['product' => $options['product']])
-        ;
+        $builder->add('cartItem', CartItemType::class, ['product' => $options['product']]);
     }
 
     /**
      * {@inheritdoc}
      */
     public function configureOptions(OptionsResolver $resolver)
     {
-        parent::configureOptions($resolver);
-
         $resolver
+            ->setDefaults([
+                'data_class' => AddToCartCommand::class,
+                'validation_groups' => ['sylius'],
```

Comment:
These should be configurable not hardcoded, why not extend `AbstractResourceType`?